src/main/java/frc/robot/commands/rider/RiderPID.java

```diff
@@ -0,0 +1,58 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands.rider;
+
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.subsystems.Rider;
+
+public class RiderPID extends Command {
+  double setPoint;
+  double TOLORANCE;
+  private Rider rider = Rider.getInstance();
+  public RiderPID(double setPoint,double TOLORANCE) {
+    this.setPoint=setPoint;
+    this.TOLORANCE=TOLORANCE;    
+ requires(rider);
+ rider.setPointRider(setPoint);
```

Comment:
Why is set setpoint in constructor and not in initialize?

Reply:
זה לא משנה?

Reply:
אחד רץ שאני קורא שאני עושה את הפעולה של הnew והשני רק שאני קורא לקומנדס 
אבל זה לא משנה כי יש לי טולורנס קבוע וגם סט פוינט אז שעני עושה בoi את הפעולה של הnew אני מגדיר אותם ואז אין שום הבדל